docs/user/integration/dynatrace/README.md

```diff
@@ -176,14 +176,48 @@ To start ingesting custom spans and Istio spans, you must enable the Istio traci
+Depending on they way your applications emit metrics, choose one of the following approaches to ingest custom metrics to Dynatrace:
 
-  > [!NOTE]
-  > The Dynatrace OTLP API does [not support](https://docs.dynatrace.com/docs/shortlink/opentelemetry-metrics-limitations#limitations) metrics in "cumulative" aggregation temporality (default temporarilty in OTel). The metrics need a custom transformation in case they are not emmitted already in the expected "delta" temporarility. A MetricPipeline does not support these transformation features.
+- Use a MetricPipeline together with a custom OTel Collector Deployment.
+
+  This approach adds the required "cumulative to delta" transformation by running an additional custom OTel Collector. The Telemetry Metric gateway is configured to ship the metrics to the custom collector, and the collector transforms them before shipping the data to the Dynatrace endpoint. This approach enables support for all metric types and inputs for the MetricPipeline. However, you must operate the additional OTel Collector in a custom way.
+
+  1. Deploy the custom OTel Collector using Helm with a custom [values.yaml](https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/dynatrace/exporter-values.yaml):
+
+        ```bash
+        helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+        helm repo update
+
+        helm upgrade --install -n ${DYNATRACE_NS} dynatrace-exporter open-telemetry/opentelemetry-collector -f https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/dynatrace/exporter-values.yaml
+        ```
+
+  1. Deploy the MetricPipeline that ships to the custom OTel Collector:
+
+        ```bash
+        cat <<EOF | kubectl apply -f -
+        apiVersion: telemetry.kyma-project.io/v1alpha1
+        kind: MetricPipeline
+        metadata:
+            name: dynatrace
+        spec:
+            input:
+                istio:
+                    enabled: true
+                prometheus:
+                    enabled: true
+            output:
+                otlp:
+                    endpoint:
+                        value: http://dynatrace-exporter-opentelemetry-collector.${DYNATRACE_NS}:4317
+        EOF
+        ```
+
+- If your application pushes metrics in delta temporality with OTLP, you can use a MetricPipeline to push metrics directly.
 
-  Use this setup when your application pushes metrics to the telemetry metric service natively with OTLP, and if you have explicitly enabled "delta" aggregation temporality. You cannot enable additional inputs for the MetricPipeline.
+  Use this setup when your application pushes metrics to the telemetry metric service natively with OTLP, and if you have explicitly enabled "delta" aggregation temporality. You cannot enable additional inputs for the MetricPipeline as these will produce metrics with "cumulative" temporarility.
```

Comment:
We can drop a few words because they just reiterate the "If..." condition above.
```suggestion
  To use this setup, you must explicitly enable the "delta" aggregation temporality. You cannot enable additional inputs for the MetricPipeline because these produce metrics with "cumulative" temperability.
```